filedialogs/filedialogs.py

```diff
@@ -50,7 +50,9 @@ def open_file_dialog(
     if ext is None:
         ext = "All Files\0*.*\0"
     else:
-        ext = "".join([f"{name}\0*.{extension}\0" for name, extension in ext])
+        extFilter = ""
```

Comment:
Please use `snake_case` for variable names, since that's the python convention. 

Also, the variable name should still be `ext`, since `extFilter` is not set when `ext=None`, leading to an `UnboundLocalError`.